backend/service/aws/dynamodb.go

```diff
@@ -118,11 +118,34 @@ func newProtoForIndexStatus(s types.IndexStatus) dynamodbv1.GlobalSecondaryIndex
 	return dynamodbv1.GlobalSecondaryIndex_Status(value)
 }
 
-func newProtoForBillingMode(s types.BillingMode) dynamodbv1.Table_BillingMode {
-	value, ok := dynamodbv1.Table_BillingMode_value[string(s)]
+// manually check the billing mode by inferring it from throughput
+// to cover cases where AWS does not return the mode in the table description
+// if a table is PROVISIONED, it will have at least 1 RCU/WCU provisioned
+// if a table is PAY_PER_REQUEST (on demand), it will have 0 RCU/WCU provisioned
+func getBillingMode(t *types.ProvisionedThroughputDescription) types.BillingMode {
+	if (*t.ReadCapacityUnits > 0) || (*t.WriteCapacityUnits > 0) {
+		return types.BillingModeProvisioned
+	}
+	if (*t.ReadCapacityUnits == 0) || (*t.WriteCapacityUnits == 0) {
+		return types.BillingModePayPerRequest
+	}
+
+	return "UNSPECIFIED" // unable to infer what billing mode it is
```

Comment:
Do we not have a type for this one?

Reply:
This check happens before the json response is marshaled into our proto, so I had to use the AWS type enums here since I'm returning an AWS type. [AWS BillingMode enum](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/dynamodb@v1.5.1/types#BillingMode) only has `provisioned` and `pay_per_request` ): 

In theory this check shouldn't ever return UNSPECIFIED because that would mean we're missing both `BillingMode` and `ProvisionedThroughput` from AWS's DescribeTable response

Reply:
Alright lets just move this to a const then